oak/proto/application.proto

```diff
@@ -94,6 +94,11 @@ message GrpcClientConfiguration {
   // The endpoint address of the external gRPC service.
   // `address` is represented as an "ip_address:tcp_port" string.
```

Comment:
What's the relationship between `address` and hostname?  And is this really restricted to IP:port?  (Also, what would an IPv6 address look like here?)  ([Earlier discussion](https://github.com/project-oak/oak/pull/640#discussion_r385267228))